Essentials/src/main/java/com/earth2me/essentials/Mail.java

```diff
@@ -0,0 +1,112 @@
+package com.earth2me.essentials;
+
+import com.earth2me.essentials.messaging.IMessageRecipient;
+import com.google.gson.JsonArray;
+import com.google.gson.JsonElement;
+import com.google.gson.JsonObject;
+import com.google.gson.JsonParser;
+
+import java.text.SimpleDateFormat;
+import java.util.Date;
+
+import static com.earth2me.essentials.I18n.tl;
+
+public class Mail {
+    private final transient IEssentials ess;
+    private final transient JsonParser jsonParser = new JsonParser();
+    private final transient ThreadLocal<SimpleDateFormat> df = ThreadLocal.withInitial(() -> new SimpleDateFormat("yyyy/MM/dd HH:mm"));
+
+    public Mail(IEssentials ess) {
+        this.ess = ess;
+    }
+
+    public void sendMail(IUser user, IMessageRecipient sender, String message) {
```

Comment:
I'd prefer we avoid `IMessageRecipient` - it's part of a messaging abstraction that was never actually implemented and should probably be phased out. I'd rather we just use `IUser`, or if absolutely necessary, add our own `MailRecipient` interface (not sure what the benefit is when all message recipients will be users though).

Reply:
> not sure what the benefit is when all message recipients will be users though

Console, while they cannot receive mail, can still send it, So IUser isn't acceptable here. I'll convert this to a custom interface tho